src/main/java/seedu/address/model/person/HighImportance.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+public class HighImportance {
+    public static final HighImportance HIGH_IMPORTANCE = new HighImportance(true);
+    public static final HighImportance NOT_HIGH_IMPORTANCE = new HighImportance(false);
+    public static final String MESSAGE_CONSTRAINTS =
+            "HighImportance should be either true or false, and should not be blank.";
+    public static final String TRUE_STRING = "true";
+    public static final String FALSE_STRING = "false";
```

Comment:
I think it's great that you're trying to reduce magic literals in our codebase! This was my mistake in the Favourites component, where I compared using the string literal "true" or "false" instead of using constants.

Since `false` and `true` are part of Java's boolean types, we should leverage on their given primitive type to derive the string representation of these types instead of creating our own constants! This way, we can reduce the number of constants we have.

For example, `String.valueOf(false)`.

Reply:
Thanks for the feedback! Instead of doing `"true"` or `"false"`, I decided to implement `String.valueOf(true)` and `String.valueOf(false)`